lib/ip_util.c

```diff
@@ -149,6 +149,55 @@ int rc_get_srcaddr(struct sockaddr *lia, const struct sockaddr *ria)
 	return 0;
 }
 
+/** Parse bindaddr config parameter to retrieve the source address
+ *
+ * bindaddr parameter in config file can have values: * or <ipv4>,<ipv6>
+ *   "192.168.11.11,2001::2" or "*,2001::2" or "192.168.11.11,*" or "*" or "192.168.11.11"
+ *   * - means the IP packet can have any outbound interface IP. 
+ *
+ * @param bindaddr_str bindaddr parameter from config file.
+ * @param srctxtaddr out paramter which will have IPv4 or IPv6 source address
+ * @param addr_type AF family type to be retrieved.
+ */
+void rc_get_source_ip(char *bindaddr_str, char *srctxtaddr, unsigned short addr_type)
+{
```

Comment:
It would be good for this function to be able to handle and notify about errors. How would a caller get notified and act on parsing or other errors found by this function?